pom.xml

```
@@ -22,6 +22,9 @@
         <!-- maven plugin versions -->
         <maven-compiler-plugin.version>3.6.0</maven-compiler-plugin.version>
         <maven-surefire-plugin.version>2.19.1</maven-surefire-plugin.version>
+
+        <!--this is used by logback-test.xml-->
+        <logRootLevel>info</logRootLevel>
```

If it affects just the test root logger, better change a little bit the name
```suggestion
        <testLogRootLevel>info</testLogRootLevel>
```